src/ui/internal-page/LegacyAdminConsoleMenuContainer.js

```diff
@@ -273,37 +274,39 @@ const LegacyAdminConsoleMenuBody = ({ userPermissions, intl, history }) => (
           <SecondaryItem
             id="menu-users"
             title={intl.formatMessage({ id: 'menu.users', defaultMessage: 'Management' })}
-            href={adminConsoleUrl('do/User/list.action')}
-            onClick={() => {}}
-          />
-          <SecondaryItem
-            id="menu-roles"
-            title={intl.formatMessage({ id: 'menu.roles', defaultMessage: 'Roles' })}
-            href={adminConsoleUrl('do/Role/list.action')}
-            onClick={() => {}}
-          />
-          <SecondaryItem
-            id="menu-groups"
-            title={intl.formatMessage({ id: 'menu.groups', defaultMessage: 'Groups' })}
-            href={adminConsoleUrl('do/Group/list.action')}
-            onClick={() => {}}
-          />
-          <SecondaryItem
-            id="menu-profile"
-            title={intl.formatMessage({ id: 'menu.profileTypes', defaultMessage: 'Profile Types' })}
-            href={adminConsoleUrl('do/Entity/initViewEntityTypes.action?entityManagerName=UserProfileManager')}
-            onClick={() => {}}
-          />
-          <SecondaryItem
-            id="menu-user-restrictions"
-            title={intl.formatMessage({ id: 'menu.users.restrictions', defaultMessage: 'Restrictions' })}
-            href={adminConsoleUrl('do/User/systemParams.action')}
-            onClick={() => {}}
+            onClick={() => history.push(ROUTE_USER_LIST)}
           />
+          {hasAccess(ROLE_SUPERUSER, userPermissions) && (
```

Comment:
I think users with following privileges can also view this menu item.
`CRUD_USERS_PERMISSION`
`VIEW_USERS_AND_PROFILES_PERMISSION`
`EDIT_USER_PROFILES_PERMISSION`

Reply:
If user has at least one of those 4 (including superuser priv) privileges, they should see list of users menu item